understanding/20/section-headings.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Section Headings</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Users understand how content is organized in sections</dd>
-         <dt>Author task</dt><dd>Where content is organized in sections, provide section headings</dd>
+         <dt>Goal</dt><dd>Users understand how content is organized in sections.</dd>
+         <dt>What to do</dt><dd>Where content is organized in sections, provide section headings.</dd>
+         <dt>Why it's important</dt><dd>Users with reduced sight or cognition can orient themselves.</dd> 
```

Comment:
@rainbreaw commented
> 'Reduced cognition' implies that there is equivalence between cognitive disabilities and intellect. Suggest rewording to
```suggestion
         <dt>Why it's important</dt><dd>Users with reduced sight or cognitive disabilities can orient themselves.</dd> 
```

Reply:
Going to invert this for readability
```suggestion
         <dt>Why it's important</dt><dd>Users with cognitive disabilities or reduced sight can orient themselves.</dd> 
```